prog/download_boot_image.rb

```diff
@@ -100,6 +100,7 @@ def sha256_sum
       ["postgres16-lantern-ubuntu-2204", "x64", "20250103.1.0"] => "bfb56867513045bc88396d529a3cc186dc44ba4d691acb51dbf45fc5a0eeb7e6",
       ["postgres17-lantern-ubuntu-2204", "x64", "20250103.1.0"] => "a95b2e5d03291783dc1753228d7a87949257a06c7b1eca2c94502ab21ffdecdb",
       ["ai-ubuntu-2404-nvidia", "x64", "20250301.1.0"] => "6392a94b248c5e299325f3687a5f48ae3a50cf5a10d54efd2be3bbea89f2f634",
+      ["ai-model-dummy", "x64", "20250312.1.0"] => "ee37cfff0d918067e0a2d762e2168d9d9ba4c604e81b9b38bb19068d83470771",
```

Comment:
`dummy` sounds like it's used for testing, etc. Maybe `ai-model-empty` or `ai-model-no-model` or `ai-modelless` 🤔